tools/cpplint.py

```diff
@@ -3089,6 +3118,14 @@ def main():
                                          codecs.getwriter('utf8'),
                                          'replace')
 
+
+  ch = logging.StreamHandler(sys.stdout)
+  logger.addHandler(ch)
+  logger.setLevel(logging.INFO)
+
+  if _cpplint_state.output_format == 'tap':
+    logger.info('TAP Version 13')
```

Comment:
As per the [spec](https://testanything.org/tap-version-13-specification.html), it should have been `version`, no?